scripts/release.sh

```diff
@@ -342,3 +341,14 @@ function branch_release() {
       ${commitish} \
       ${TAG}
 }
+
+function branch_release() {
+    local tags="$(git tag | cut -d 'v' -f2 | cut -d '.' -f1-2 | sort | uniq )"
+    local branches="$( { (git branch -r | grep origin/release-) ; (git branch  | grep release-); } | cut -d '-' -f2 | sort | uniq)"
+    if [ "$tags" == "$branches" ]
+    then
+        echo "Everything is released. Nothing to do!"
```

Comment:
The no-op should come before the build itself. This way you're going through all the build process, but just not creating the release on GitHub.